packages/ui/src/components/widgets/WidgetContainer.tsx

```diff
@@ -69,65 +69,50 @@ const WidgetContainer: FC<WidgetContainerProps> = ({
   const [addressModalOpen, setAddressModalOpen] = useState(false)
   return (
     <div className="relay-kit-reset">
-      <Flex
-        direction="column"
-        css={{
-          width: '100%',
-          borderRadius: 'widget-border-radius',
-          overflow: 'hidden',
-          backgroundColor: 'widget-background',
-          boxShadow: 'widget-box-shadow',
-          border: 'widget-border',
-          p: '4',
-          minWidth: 300,
```

Comment:
Did we lose the midWidth and maxWidth?

Reply:
The minWidth and maxWidth container just got moved to the SwapWidget and ChainWidgets